homeassistant/components/smarty/switch.py

```diff
@@ -0,0 +1,93 @@
+"""Support for Salda Smarty XP/XV Ventilation Unit Sensors."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+import logging
+from typing import Any
+
+from pysmarty2 import Smarty
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .coordinator import SmartyConfigEntry, SmartyCoordinator
+from .entity import SmartyEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True, kw_only=True)
+class SmartySwitchDescription(SwitchEntityDescription):
+    """Class describing Smarty switch."""
+
+    is_on_fn: Callable[[Smarty], bool]
+    turn_on_fn: Callable[[Smarty], bool | None]
+    turn_off_fn: Callable[[Smarty], bool | None]
+
+
+ENTITIES: tuple[SmartySwitchDescription, ...] = (
+    SmartySwitchDescription(
+        key="boost",
+        translation_key="boost",
+        is_on_fn=lambda smarty: bool(smarty.boost),
```

Comment:
is the extra `bool()` needed?

Reply:
Removed as smarty.boost is already bool.